### PR TITLE
PIL-221-Fixed Acceptance test

### PR DESCRIPTION
**Pull request by AKKUM**

### CODE REVIEW

File: src/test/resources/features/CheckAnswers.feature [hunk 1]
```diff
@@ -5,4 +5,7 @@ Feature: Check Answers Page
     Then The Heading should be Welcome to OECD Pillar Two
     Then The Body content should be Welcome to your new frontend. Please see the README file for a guide to getting started.
     When I click Continue button
+    Then The Heading should be Do you have trading business address in UK?
+    Then The Body content should be Do you have trading business address in UK?
+    And I select value_0 and continue
```
Comment (srikanthmeurostar) [inline src/test/resources/features/CheckAnswers.feature:10]: can we provide the option text here instead value_0

File: src/test/resources/features/TradingBusiness.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@tests
+Feature: Trading Business confirmation Page
+  Scenario: 1 - Display trading Business confirmation Page
+    Given Organisation User logs in to register for Pillar2
+    Then The Heading should be Welcome to OECD Pillar Two
+    Then The Body content should be Welcome to your new frontend. Please see the README file for a guide to getting started.
+    When I click Continue button
+    Then The Heading should be Do you have trading business address in UK?
+    Then The Body content should be Do you have trading business address in UK?
+    And I select value_0 and continue
```
Comment (srikanthmeurostar) [inline src/test/resources/features/TradingBusiness.feature:10]: same here

**Review by srikanthmeurostar — CHANGES_REQUESTED** [review]

File: src/test/resources/features/CheckAnswers.feature (hunk 1 above)
Comment (srikanthhmrc) [inline src/test/resources/features/CheckAnswers.feature:10]: can we mention the option to be selected instead value_0

File: src/test/resources/features/TradingBusiness.feature (hunk 2 above)
Comment (srikanthhmrc) [inline src/test/resources/features/TradingBusiness.feature:10]: same here

**Review by nitin-srivastav — APPROVED** [review]

**Pull request merged**